docs/data-example.rst

```diff
@@ -21,7 +21,12 @@ and as CSV:
 .. jupyter-execute::
     :hide-code:
 
-    print(db['files'].get().to_csv())
+    df = db['files'].get()
+    # Since pandas 1.4.0 DataFrame.to_csv()
+    # no longer works for categories with dtype Int64
+    # we have to convert column to plain Int64
+    df['label_map_int'] = df['label_map_int'].astype('Int64')
```

Comment:
This doesn't look to me like something a user should have to care about.

Reply:
Maybe we should first open an issue at `pandas` and ask if the behavior is desired or a bug?

Reply:
A user will call `db.save()` to save the whole database or `db['table'].save()` to save a particular table.

Reply:
OK, I now realized that this code is not visible for the user, and we just need it to show the CSV output:

![image](https://user-images.githubusercontent.com/173624/163972270-c96ffbf1-646d-4a4c-a325-83142c179ba9.png)

So, it's still nasty that we need this workaround, but it should be fine as the user never has to call this.